packages/app-builder/src/services/validation/scenario-validation-error-messages.ts

```diff
@@ -9,7 +9,7 @@ import { assertNever } from 'typescript-utils';
 // Edit this type to handle contextual data for each error code
 export type EvaluationErrorViewModel =
   | {
-      error: 'UNEXPECTED_ERROR';
+      error: 'UNEXPECTED_ERROR' | 'RUNTIME_EXPRESSION_ERROR';
```

Comment:
I consider `'RUNTIME_EXPRESSION_ERROR'` like `'UNEXPECTED_ERROR'`: meaning we will display the raw error message since those are "not qualified" errors that should rarelly hapen.